### ISSUE TITLE
[Keyboard navigation - ADF - Azure data factory] Controls are not accessible through keyboard on Azure data factory window screen.

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/AzureStorageExplorer#3846]
**User Experience:**
User will find difficulty to interact with the controls if that controls are not accessible through keyboard as user won't be able to perform functionality over such control.

**Test Environment:**
Application Version: 1.16.0-dev
Build Number: 20200921.1

**Pre-Requisite:**
Navigate to the storage expoler.exe and select it  

**Repro Steps:​**
- Launch Azure Storage Explorer app.
- Navigate to visual icon Add data factory control at left panel.
- Navigate to various control like Minimize/Maximize/Restore down/Close on Azure data factory window screen.
- Verify various control like Minimize/Maximize/Restore down/Close by using keyboard.

**Actual Result:**
Controls(Minimize/Maximize/Restore down/Close) are not accessible through keyboard on Azure data factory window screen and due to this, user is unable to interact with such control and also won't be perform functionality over such control.

**Expected Result:**
Controls(Minimize/Maximize/Restore down/Close) should be accessible through keyboard on Azure data factory window screen so that user could easily interact with such control to proceed further.

**Note:**
- Close control is also not accessible through keyboard on Add connection window screen.
- Cancel/Remove control is also not accessible through keyboard on Connection edit field of Azure data factory window.

![MAS2 1 1_Azure data factory](https://user-images.githubusercontent.com/72871089/101141111-d6d12a80-3639-11eb-87f0-148b432a5556.png)
[MAS2.1.1_Azure data factory.zip](https://github.com/microsoft/AzureStorageExplorer/files/5641485/MAS2.1.1_Azure.data.factory.zip)
[MAS2.1.1_Add connection.zip](https://github.com/microsoft/AzureStorageExplorer/files/5641487/MAS2.1.1_Add.connection.zip)
